pkg/util/validation/limits.go

```diff
@@ -474,3 +474,19 @@ func SmallestPositiveNonZeroDurationPerTenant(tenantIDs []string, f func(string)
 	}
 	return *result
 }
+
+// BiggestDurationPerTenant is returning the maximum duration
+// per tenant. Without tenants given it will return a 0 duration.
+func BiggestDurationPerTenant(tenantIDs []string, f func(string) time.Duration) time.Duration {
```

Comment:
**suggestion(optional):** I prefer MaxDurationPerTenant

```suggestion
// MaxDurationPerTenant is returning the maximum duration
// per tenant. Without tenants given it will return a 0 duration.
func MaxDurationPerTenant(tenantIDs []string, f func(string) time.Duration) time.Duration {
```

Reply:
:+1: Agreed, done that